go/vt/vtctl/workflow/utils.go

```diff
@@ -217,6 +217,31 @@ func stripTableForeignKeys(ddl string, parser *sqlparser.Parser) (string, error)
 	return newDDL, nil
 }
 
+func stripAutoIncrement(ddl string, parser *sqlparser.Parser) (string, error) {
+	ast, err := parser.ParseStrictDDL(ddl)
+	if err != nil {
+		return "", err
+	}
+
+	stripAutoIncrement := func(cursor *sqlparser.Cursor) bool {
+		switch node := cursor.Node().(type) {
+		case sqlparser.DDLStatement:
+			if node.GetTableSpec() != nil {
+				for _, column := range node.GetTableSpec().Columns {
```

Comment:
Should this care if the column is `unique` or not `unique`?

Reply:
Those considerations apply to the table schema when/where it was created in MySQL (it can only be used with a PK column, and those must be NOT NULL and unique). In our case here we are removing it and don't care.